tests/system/cli.go

```diff
@@ -263,7 +263,7 @@ func (c WasmdCli) QuerySmart(contractAddr, msg string, args ...string) string {
 }
 
 // QueryBalances queries all balances for an account. Returns json response
-// Example:`{"balances":[{"denom":"node0token","amount":"1000000000"},{"denom":"utgd","amount":"400000003"}],"pagination":{}}`
+// Example:`{"balances":[{"denom":"node0token","amount":"1000000000"},{"denom":"ustake","amount":"400000003"}],"pagination":{}}`
```

Comment:
Good 👁️ ! Please use `stake` here and everywhere else instead of `ustake`. `stake is the `sdk.DefaultBondDenom` .
```suggestion
// Example:`{"balances":[{"denom":"node0token","amount":"1000000000"},{"denom":"stake","amount":"400000003"}],"pagination":{}}`
```